### ISSUE TITLE
0.4 release: speedups don't build with Python 3.7

### ISSUE THREAD

**Issue by fkrull** [Kozea/tinycss#17]
The generated speedups.c file included in the tinycss 0.4 release on PyPI doesn't build correctly on Python 3.7. The file was generated with an old version of Cython that didn't support Python 3.7.

Ref: https://github.com/cython/cython/issues/1978
Ref: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=910751

**Comment by timokau** [comment]
For what its worth, the current workaround is to manually remove the `speedups.c` file (since otherwise cython won't regenerate it) and make sure `cython` is installed during build.

**Comment by fkrull** [comment]
Yeah, that's how we ended up solving it in the Debian package as well: https://salsa.debian.org/python-team/modules/python-tinycss/commit/632f2d4476721e3e8e606767b270cfd15b6c3c17
